test/aws-kinesis/aws-kinesis-source.feature

```diff
@@ -8,8 +8,9 @@ Feature: AWS Kinesis Kamelet
   Scenario: Create AWS Kinesis data stream
     Given variables
     | aws.kinesis.clientName | aws-kinesis-client-citrus:randomString(10, LOWERCASE) |
-    | aws.kinesis.command    | "create-stream", "--stream-name", "${aws.kinesis.stream}", "--shard-count", "1" |
+    | aws.kinesis.command    | create-stream --stream-name ${aws.kinesis.stream} --shard-count 1 |
     When load Kubernetes resource aws-kinesis-client.yaml
+    Then sleep 5000 ms
```

Comment:
is that `sleep` just for debugging reasons?

Reply:
This was added because on my env, the pods were deleted before the command could be finished. I'll remove it and in case it's observed by anyone else, we can try to find a more elegant solution.